Maestro/subscriptions.json

```
@@ -1055,15 +1055,15 @@
     },
     {
       "triggerPaths": [
-        "https://github.com/dotnet/aspnetcore/blob/release/2.1//**/*",
-        "https://github.com/dotnet/aspnetcore/blob/release/3.1//**/*",
-        "https://github.com/dotnet/aspnetcore/blob/release/5.0//**/*",
-        "https://github.com/dotnet/aspnetcore/blob/release/6.0*//**/*",
-        "https://github.com/dotnet/aspnetcore-tooling/blob/release/3.1//**/*",
-        "https://github.com/dotnet/ef6/blob/release/6.4//**/*",
-        "https://github.com/dotnet/efcore/blob/release/3.1//**/*",
-        "https://github.com/dotnet/efcore/blob/release/5.0//**/*",
-        "https://github.com/dotnet/efcore/blob/release/6.0*//**/*"
+        "https://github.com/dotnet/aspnetcore/blob/release/2.1/**/*",
+        "https://github.com/dotnet/aspnetcore/blob/release/3.1/**/*",
+        "https://github.com/dotnet/aspnetcore/blob/release/5.0/**/*",
+        "https://github.com/dotnet/aspnetcore/blob/release/6.0*/**/*",
+        "https://github.com/dotnet/aspnetcore-tooling/blob/release/3.1/**/*",
+        "https://github.com/dotnet/ef6/blob/release/6.4/**/*",
+        "https://github.com/dotnet/efcore/blob/release/3.1/**/*",
+        "https://github.com/dotnet/efcore/blob/release/5.0/**/*",
+        "https://github.com/dotnet/efcore/blob/release/6.0*/**/*"
```

The non-preview paths are working fine and the `//` part has been required to avoid triggering on e.g. temporary `darc` branches. Suggest reverting all changes except those specifically about release/6.0 here and above.
```suggestion
        "https://github.com/dotnet/aspnetcore/blob/release/2.1//**/*",
        "https://github.com/dotnet/aspnetcore/blob/release/3.1//**/*",
        "https://github.com/dotnet/aspnetcore/blob/release/5.0//**/*",
        "https://github.com/dotnet/aspnetcore/blob/release/6.0*/**/*",
        "https://github.com/dotnet/aspnetcore-tooling/blob/release/3.1//**/*",
        "https://github.com/dotnet/ef6/blob/release/6.4//**/*",
        "https://github.com/dotnet/efcore/blob/release/3.1//**/*",
        "https://github.com/dotnet/efcore/blob/release/5.0//**/*",
        "https://github.com/dotnet/efcore/blob/release/6.0*/**/*"
```